lib/agenda/check-url.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+/**
+ * Given a mongo connection url the function will check with a regex if it is valid
+ * @param url
+ * @returns {AggregationCursor|*|RegExpMatchArray|Promise<Response | undefined>}
```

Comment:
I think this should return boolean?

Reply:
This was automatically generated when I created the docblock, technically it doesnt return a boolean when it is true

Reply:
Since this is now a separate function, I think it might be clearer to call it `needsMongoProtocol` and always return a boolean?

Reply:
Aah I like that actually, good idea, as the function checks if it has the mongoProtocol, I have called it hasMongoProtocol